classes/session.php

```diff
@@ -332,12 +322,11 @@ public static function delete_flash($name = null)
 	/**
 	 * create a new session
 	 *
-	 * @access	public
 	 * @return	void
 	 */
 	public static function create()
 	{
-		return static::instance()->create();
+		static::instance()->create();
```

Comment:
This could possibly cause a BC issue if someone is relying on the return value from `create()` from a custom session driver.